### ISSUE TITLE
QuarterlyGradingPmtInfo - LAND_USE_AF No Values Populating in Export

### ISSUE THREAD

**Issue by JuanitaMiles** [kcigeospatial/balt_co_ETL#20]
Steps to Reproduce:
Reporting Year = 2017
On Failure = Continue on Errors
Current User = KCI\Juanita.Miles

No values are being populated the MDE Export, QuarterlyGradingPmtInfo.LAND_USE_AF.

Source (QRT_GRADING_INFO.LAND_USE_AF):
![image](https://user-images.githubusercontent.com/31770504/31451043-05a08f6e-ae79-11e7-9e31-9b7d76bf468e.png)
MDE Export (QuarterlyGradingPmtInfo.LAND_USE_AF):
![image](https://user-images.githubusercontent.com/31770504/31451083-1fe5eba8-ae79-11e7-9ca7-e3d2728984b5.png)

MDE Export should populate with the MDPLandUse Domain Descriptions.
13 --> **High Density Residential**
17 --> **Extractive**
14 --> **Commercial**
13 --> **High Density Residential**
15 --> **Industrial**

**Comment by gerrykelly** [comment]
@JuanitaMiles 
I don't see that this comes from any issue with the ETL code.  I'm not sure why it is happening so I'll turn this back to you with a test FGDB and some info to see if you can catch anything at the ArcGIS level.

The FGDB is \\vm-wsscmta\MDEdbExports\LAND_USE_AF__test.gdb

The step in the ETL that should populate LAND_USE_AF is when QuarterlyGradingPmtInfo_p gets appended to the QuarterlyGradingPmtInfo table.

You'll see that the AF & BF field types (that started as text at the county) are both correct:
![image](https://user-images.githubusercontent.com/8259136/31461487-a265763e-ae97-11e7-878b-278ef4a3cb83.png)

If you went to ArcMap you'd see that the fields show like this ...
![image](https://user-images.githubusercontent.com/8259136/31461501-afa81a18-ae97-11e7-9f18-037c506d05c9.png)
![image](https://user-images.githubusercontent.com/8259136/31461507-b4400234-ae97-11e7-999b-d8a0faa2d183.png)

and when you show codes instead, like this:
![image](https://user-images.githubusercontent.com/8259136/31461539-d1cfddb0-ae97-11e7-988f-427f6475f47f.png)

In the ETL, the Append TEST works, so I don't see why BF gets populated and AF does not.
If you manually in ArcCatalog do the append you get the same result.

Looking at NO_TEST, the fields map so it looks like it should work
![image](https://user-images.githubusercontent.com/8259136/31461611-117459a0-ae98-11e7-958e-e0a24f4c9409.png)

And no error are returned in the geoprocessing results:
![image](https://user-images.githubusercontent.com/8259136/31461692-4167a202-ae98-11e7-8f7b-a345e56c5790.png)


Let me know if you can find anything

**Comment by gerrykelly** [comment]
@JuanitaMiles @leeensminger 
You can still look into this if you'd like, but later I happened to catch that LAND_USE_AF is a subtype field in the MDE FGDB but not for BaltCO.  That's a schema difference and probably the cause of this.  So, I'll remove my assignment until you get direction on dealing with that (I wonder if it is even intentional by MDE in the first place)

![image](https://user-images.githubusercontent.com/8259136/31465515-31e501d0-aea2-11e7-8e84-a0ebadab6a8c.png)

**Comment by JuanitaMiles** [comment]
@gerrykelly @leeensminger 
Nice catch Gerry. I'm glad you found this, since I, to be honest, don't know how to access that FGDB you created for me...I'll put this on my list of topics for Friday (both, the schema change and my lack of knowledge...)

**Comment by JuanitaMiles** [comment]
@gerrykelly @leeensminger 
Okay so I figured out how to access the LAND_USE_AF_test.gdb (Duh). I copied it to my working folder...tried the append once with the subtype applied got the same results...then removed the subtype, ran the append again and believe it or not EVERYTHING, including all the other domains that didn't display properly from the other issue I logged, comes out perfect! 

![image](https://user-images.githubusercontent.com/31770504/31470705-25f8bc48-aeb3-11e7-8b70-3a85e339f7ef.png)

So I think we can kill two birds with one stone by hopefully removing the LAND_USE_AF subtype.
